src/fluree/db/server.clj

```diff
@@ -262,7 +262,7 @@
 
 (defn -main []
   (println "******************************************"
-           "\nFluree version" (meta/version)
+           "\nFluree version" (meta/version) "DEP test 2.0.4_dan-test"
```

Comment:
We don't want to merge that, do we?

Reply:
I couldn't figure out where to change the version and needed some way to tell if the tool was using the right image, but now that I know the right way I can ditch this.